test/KisSpec.hs

```diff
@@ -42,17 +44,37 @@ spec = do
                 pat <- req (CreatePatient $ Patient "xy")
                 void $ req (PlacePatient pat (toSqlKey 1)))
             `shouldThrow` (== ConstraintViolation)
-    describe "withSqliteKis" $ do
+    describe "withSqliteKis" $
          it "can run two clients in sequence" $
             withSqliteKis InMemory kisConfig $ \kis -> do
                 patId <- runClient kis $ req (CreatePatient $ Patient "Simon")
                 pats <- runClient kis $ req GetPatients
                 pats `shouldBe` [Entity patId (Patient "Simon")]
+    describe "withSqlitePool" $ do
+          it "can run two clients in sequence" $
+             withTempFile "/tmp/" "tmpKisDB" $ \fp _ ->
+                 withSqliteKis (PoolBackendType (T.pack fp) 10) kisConfig $ \kis ->
+                     do patId <- runClient kis $ req (CreatePatient $ Patient "Simon")
+                        pats <- runClient kis $ req GetPatients
+                        liftIO $ pats `shouldBe` [Entity patId (Patient "Simon")]
+          it "can run two clients in parallel" $
```

Comment:
Ah jetzt habe ich gesehen, dass `writeNotif` ja gar nicht den anderen Thread benachrichtigt, sondern nur in die DB schreibt, richtig? Ich sehe nicht so ganz, wer den anderen Thread dann benachrichtigt.